stage1/find-em.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/sh
+#
+# Find entries for the rkt ACI manifest files.
+# Run from within the CoreOS version of interest.
+#
+# Post prosessing tips:
+#   cat found.raw | egrep '^[^=]' | egrep --invert-match '[^[0-9]\.so$' | sort | uniq -c > found.uniq
+#   manual clanup
+#   cat found.uniq | sort | uniq > found.manifest
+#
+
+# TODO: get values from makefile
+#: ${BOARD:="@@BOARD@@"}
```

Comment:
With that we'd need the makefiles on the target.  I was thinking to use sed in the makefile to generate the script, then it will be self contained.